src/main/resources/data.sql

```diff
@@ -9,6 +10,18 @@ create table user (
     updated timestamp not null default current_timestamp()
 );
 
+create table address (
+    id int primary key AUTO_INCREMENT,
+    line1 varchar(100) not null,
+    line2 varchar(100),
+    city varchar(100) not null,
+    state varchar(100) not null,
+    zip varchar(100) not null,
```

Comment:
zip codes don't go to a length 100.

Reply:
And you had this as an int in the dao so not 100 there either.  It's nitpicky though.